### ISSUE TITLE
ArgumentException: The parameter is incorrect

### ISSUE THREAD

**Issue by SeanSobey** [madskristensen/ShowTheShortcut#7]
### Installed product versions
- Visual Studio: 2015 Enterprise
- This extension: 1.4.27

### Description
I get an exception trace in the 'Learn the Shortcut' output window instead of the usual shortcut keys.

### Steps to recreate
(Hard to replicate, but I think:)
1. Was navigating around in an embedded browser tab in Visual Studio.
2. At some point after the tab was opened I got the exception.

### Current behavior
2016/11/10 9:00:24 AM: System.ArgumentException: The parameter is incorrect. (Exception from HRESULT: 0x80070057 (E_INVALIDARG))
   at EnvDTE.Commands.Item(Object index, Int32 ID)
   at ShowTheShortcut.CommandHandler.AfterExecute(String Guid, Int32 ID, Object CustomIn, Object CustomOut) in C:\projects\showtheshortcut\src\CommandHandler.cs:line 99
2016/11/10 9:00:24 AM: System.ArgumentException: The parameter is incorrect. (Exception from HRESULT: 0x80070057 (E_INVALIDARG))
   at EnvDTE.Commands.Item(Object index, Int32 ID)
   at ShowTheShortcut.CommandHandler.AfterExecute(String Guid, Int32 ID, Object CustomIn, Object CustomOut) in C:\projects\showtheshortcut\src\CommandHandler.cs:line 99

**Comment by jcansdale** [comment]
Me too. It looks like some commands are being executed that don't exist in the `Commands` collection. I wonder what that's all about? It would be interesting to dump the `Guid`/`ID` to see what they are.

```
01/06/2017 09:18:24: System.ArgumentException: The parameter is incorrect. (Exception from HRESULT: 0x80070057 (E_INVALIDARG))
   at EnvDTE.Commands.Item(Object index, Int32 ID)
   at ShowTheShortcut.CommandHandler.AfterExecute(String Guid, Int32 ID, Object CustomIn, Object CustomOut) in C:\projects\showtheshortcut\src\CommandHandler.cs:line 101
```

**Comment by RachelDavids** [comment]
Likewise - getting the same issue when typing into the Command Window. (VS2017 15.6.4)

**Comment by TobiasSekan** [comment]
Same here on Windows 8.1 - Visual Studio 2017 15.7.0 Preview 5.0
```
30.04.2018 14:43:34: System.ArgumentException: Falscher Parameter. (Ausnahme von HRESULT: 0x80070057 (E_INVALIDARG))
   bei EnvDTE.Commands.Item(Object index, Int32 ID)
   bei ShowTheShortcut.CommandHandler.AfterExecute(String Guid, Int32 ID, Object CustomIn, Object CustomOut) in C:\projects\showtheshortcut\src\CommandHandler.cs:Zeile 99.
```

**Comment by jcansdale** [comment]
This has been fixed in #13. You can find the VSIX here:
https://ci.appveyor.com/api/buildjobs/90n2c847g4592nvt/artifacts/ShowTheShortcut.vsix

@SeanSobey @madskristensen can we clone this issue?

**Comment by madskristensen** [comment]
I assume you meant "close" :)
